.github/workflows/upload.yml

```diff
@@ -22,7 +22,7 @@ jobs:
 
       - name: Install test dependencies
         run: |
-          pip install wheel pytest pytest-cov pytest-mock --upgrade
+          pip install wheel pytest pytest-cov pytest-mock flaky --upgrade

```

Comment:
Since the shared device test suite depends on it. This was stopping the upload to PyPI action from working.